account-kit/infra/src/client/smartAccountClient.test.ts

```diff
@@ -85,6 +77,48 @@ describe("AlchemySmartAccountClient tests", () => {
     );
   });
 
+  it("should set the headers with tracking", async () => {
+    const client_ = givenClient();
+    const client = clientHeaderTrack(
+      clientHeaderTrack(client_, "test"),
+      "afterTest"
+    );
+    await client
+      .request({ method: "eth_supportedEntryPoints", params: [] })
+      .catch(() => {});
+
+    console.log(
+      "BLUJ ",
+      JSON.stringify(
+        fetchSpy.mock.calls.map((x) => x[1]?.headers),
+        null,
+        2
+      )
+    );
```

Comment:
```suggestion
```

Reply:
Thank you, I missed this. This was the point of the BLUJ, to be easy to remove, shame on me...